noxfile.py

```diff
@@ -24,7 +24,7 @@
 GOOGLE_AUTH = 'google-auth >= 0.10.0'
 
 
-@nox.session(python=['2.7', '3.4', '3.5', '3.6', '3.7'])
+@nox.session(python=['3.5', '3.6', '3.7'])
```

Comment:
Would you mind dropping 3.8 in here as well?

Reply:
Done.

I also bumped the Python versions in other sessions to `3.8`, as I believe this a standard practice (running misc. checks with the latest supported version).